Installing-and-Configuring/Building.md

```diff
@@ -2,29 +2,39 @@
 layout: default
 title: Building Heads
 permalink: /Building/
-nav_order: 8
+nav_order: 2
+parent: Installing and configuring
 ---
 
 Building Heads
 ===
-Heads is supposed to be a [reproducible build](https://reproducible-builds.org/) and as of [v0.1.0](https://github.com/osresearch/heads/releases/tag/v0.1.0) it achieved this goal.  The downside is that the initial build can take a very long time as it downloads and builds all of the its dependencies.  One issue right now is that it builds not just one, but *two* cross compilers and as a result takes about 45 minutes.  Luckily subsequent builds only take about 30 seconds to produce a full coreboot and Linux ROM image, but that first ones a doozy...
+
+Heads is supposed to be a [reproducible build](https://reproducible-builds.org/)
+ and as of [v0.1.0](https://github.com/osresearch/heads/releases/tag/v0.1.0) it
```

Comment:
Would remove release related information
Still say that it is a goal of heads to be reproducible where tags are pointed to stoppers, to trigger community developers involvement, and point to https://circleci.com/gh/osresearch/heads for Ci builded ROMs.